app/src/main/java/com/winlator/cmod/XrAPI.java

```diff
@@ -42,6 +47,63 @@ public XrAPI(String path) throws Exception {
                 throw new Exception("Filesystem issue");
             }
         }
+
+        //Set Debug directory
+        File debugDir = new File(DEFAULT_DEBUG_PATH);
```

Comment:
As we discussed in chat, let's not add it into a constructor but as a new method called from XrActivity (commented out by default).